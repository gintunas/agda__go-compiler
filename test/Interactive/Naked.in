λ issue5132 → issue5132
:metas
:quit
