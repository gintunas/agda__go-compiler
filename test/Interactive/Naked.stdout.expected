λ issue5132 → issue5132
Sort _2
_3 : _2
